app/controllers/sessions_controller.rb

```diff
@@ -0,0 +1,24 @@
+class SessionsController < ApplicationController
+  def new; end
+
+  def create
+    user = User.find_by(email: session_params['email'])  
+    if user && user.authenticate(session_params['password']) 
```

Comment:
1. Use symbol syntax
```suggestion
    if user && user.authenticate(session_params[:password]) 
```
2. Use `&.` operator